src/Library/demos/Drawing Area/main.js

```diff
@@ -0,0 +1,86 @@
+import Cairo from "cairo";
+import Gtk from "gi://Gtk?version=4.0";
+Gtk.init();
+
+const drawingArea = workbench.builder.get_object("draw");
+const scaleRotate = workbench.builder.get_object("rotate");
```

Comment:
no need to rename stuff, it's mental overload

```suggestion
const drawing_area = workbench.builder.get_object("draw");
const scale = workbench.builder.get_object("scale");
```